### ISSUE TITLE
Read history

### ISSUE THREAD

**Issue by MrTaylab** [quran/quran_android#2871]
It would be really nice if we can set how many bookmarks should be saved in history.
Currently, the last three readings are saved and can be selected. But 3 are very few

**Comment by MahmoudMabrok** [comment]
I think 3 is fine, if need more we can mark as bookmark.

**Comment by ahmedre** [comment]
agreed - we're thinking about how to make the bookmarking experience consistent between iOS and Android also, and how to bring in something like khatmas, which would solve at least some of these use cases.
